infoschema/tables.go

```diff
@@ -1578,7 +1578,7 @@ func (it *infoschemaTable) RemoveRecord(ctx sessionctx.Context, h kv.Handle, r [
 }
 
 // UpdateRecord implements table.Table UpdateRecord interface.
-func (it *infoschemaTable) UpdateRecord(ctx sessionctx.Context, h kv.Handle, oldData, newData []types.Datum, touched []bool) error {
+func (it *infoschemaTable) UpdateRecord(gctx context.Context, ctx sessionctx.Context, h kv.Handle, oldData, newData []types.Datum, touched []bool) error {
```

Comment:
Better  use  `ctx` for `context.Context` and `sctx` for `sessionctx.Context`, the former is a standard one.

Reply:
Please address the comment or reply it.